docs/tutorial/start.md

```diff
@@ -0,0 +1,47 @@
+# Pierwsze uruchomienie serwera
+
+Przy zakupie serwera, do wyboru będzie silnik serwera i jego wersja. Przy poprawnym wyborze wersji, serwer po zakupie powinien włączyć sie automatycznie. W razie problemów zapytaj na naszym [discordzie](https://discord.gg/XNHmCUK2uQ).<br />
+
+Poniżej w odpowiednich sekcjach zostanie pokazane: <br />
+* Jak wgrać plugin na serwer,
+* Jak wgrać mody na serwer,
+* Jak zmienić seed świata,
+* Jak wgrać swój świat.
+
+W prawym rogu znajdują sie wszystkie sekcje, można wygodnie przeskoczyć do tej w której jesteś zainteresowany.
+
+## Jak wgrać plugin na serwer
+
+Na samym początku by móc wgrać plugin na serwer musisz go pobrać, interesujące cie pluginy można znależć na [https://www.spigotmc.org](https://www.spigotmc.org). <br />
+
+By wrzucić plugin musisz wejść do folderu `plugins` z głównego katalogu, przedstawionym na zdjęciu poniżej. ![](/img/start/mainfolderpaper.png)
+
+Znajdując się już w folderze plugins przeciągamy plik pluginu z eksploratora plików na strone, a po chwili, plugin znajdzie sie na naszym serwerze.
+
+Do rozpoczęcia działania pluginu potrzebujemy zrestartować swój serwer.
+
+
+## Jak wgrać mody na serwer
+
+Na samym początku by móc wgrać mody na serwer musisz go pobrać, interesujące cie mody można znależć na [https://www.curseforge.com/](https://www.curseforge.com/minecraft/mc-mods). <br />
+
+By wrzucić plugin musisz wejść do folderu `mods` z głównego katalogu, przedstawionym na zdjęciu poniżej. ![](/img/start/mainfolderforge.png)
+
+Znajdując się już w folderze mods przeciągamy plik moda z eksploratora plików na strone, a po chwili, mod znajdzie sie na naszym serwerze.
+
+Do rozpoczęcia działania moda potrzebujemy zrestartować swój serwer.
+
+## Jak zmienić seed świata
+
+:::warning Uwaga!
+Zmiana tych ustawień wiąże się z usunięciem obecnego świata, co spowoduje usunięcie obecnego postępu.
```

Comment:
Zmiana samego seeda nie powoduje usuniecia swiata, tylko w poradniku zalecasz usuniecie wiec doprecyzowalbym to.
```suggestion
Zmiana tych ustawień zgodnie z poniższym opisem wiąże się z usunięciem obecnego świata, co spowoduje usunięcie obecnego postępu.
```